doc/api/api_changes/2017-06-08-PGE.rst

```diff
@@ -0,0 +1,4 @@
+Added support for RGB(A) images in pcolorfast
+`````````````````````````````````````````````
+
+pcolorfast now excepts 3D images (x,y,color) as inputs in `C` if they qualify for pcolorimage or image
```

Comment:
"excepts..." -> "accepts 3D (RGB or RGBA) arrays if the X and Y specifications allow image or pcolorimage rendering; they remain unsupported by the more general quadmesh rendering."